crates/cli/src/fs.rs

```diff
@@ -1,13 +1,25 @@
-use std::{io, os, path::Path};
+use std::{io, path::Path};
+
+#[cfg(windows)]
+const ERROR_PRIVILEGE_NOT_HELD: i32 = 1314;
 
 #[cfg(unix)]
 pub fn symlink_dir(original: &Path, link: &Path) -> io::Result<()> {
-    os::unix::fs::symlink(original, link)
+    std::os::unix::fs::symlink(original, link)
 }
 
 #[cfg(windows)]
 pub fn symlink_dir(original: &Path, link: &Path) -> io::Result<()> {
-    os::windows::fs::symlink_dir(original, link)
+    match std::os::windows::fs::symlink_dir(original, link) {
+        Ok(_) => Ok(()),
+        Err(e) => {
+            match e.raw_os_error() {
+                // If we don't have the privilege to create a symlink, we can try to create a junction instead.
+                Some(ERROR_PRIVILEGE_NOT_HELD) => junction::create(original, link),
+                _ => Err(e),
+            }
+        }
+    }
```

Comment:
Just use junctions on Windows. It is not optimal to always try a symlink and then retry a junction. It will be a lot more file system operations.